platform/src/Playground.js

```diff
@@ -289,12 +287,14 @@ function initializePanels() {
         }
     });
 
+    /* TODO Fix run program shortcut key original epsilon playground behaviour.
```

Comment:
It is broken functionality from the initial refactoring of the epsilon playground that assumes a fixed action button rather than the flexibly panels and buttons of the education platform. We can now have multiple action buttons so we nee d to add a way for a teacher to specify any shortcuts in the activity config.

Reply:
Thanks. Would be good to add an issue for this.

Reply:
Raised https://github.com/mdenet/educationplatform/issues/182